homeassistant/components/uvc/camera.py

```diff
@@ -109,10 +110,28 @@ def supported_features(self):
 
         return 0
 
+    @property
+    def state_attributes(self):
+        """Return the camera state attributes."""
+        attr = super().state_attributes
```

Comment:
This isn't needed. The state attributes will be merged with the device state attributes in the Entity base class.